docs/console/project-configuration/kustomize-your-configurations/kustomize-use-cases.md

```diff
@@ -143,4 +144,8 @@ patches:
     annotationSelector: myAnnotation=annotationValue,otherAnnotation=otherValue
 ```
 
-With the example above, the `istio-proxy` container will be add to deployments with **all the specified labels and annotations**.
+With the example above, the `istio-proxy` container will be added to deployments with **all the specified labels and annotations**.
+
+:::info
+In case of On-Premise installation, you can [add default labels and annotations](/marketplace/add_to_marketplace/contributing_overview.md#common-to-microservice-items-plugins-templates-examples) to the items since you have the full control of the Marketplace.
```

Comment:
```suggestion
In case of On-Premise installation, instead of using kustomize, you can [add default labels and annotations](/marketplace/add_to_marketplace/contributing_overview.md#common-to-microservice-items-plugins-templates-examples) directly to the items since you have the full control of the Marketplace.
```